models/exchange.yaml

```diff
@@ -24,6 +24,10 @@ properties:
     description: Name of the exchange.
     type: string
     opencrypto-weight: 1
+  platform:
+    title: Platform
+    description: Platform used for the exchange.
+    type: string
```

Comment:
Add please pattern for validation:
```yaml
platform:
  ...
  pattern: '^[a-z0-9-:]+$'
```

This will allow in this property only small letters `a-z`, numbers `0-9`, dash `-` and colon `:`